frontend/src/metabase/entities/users/forms.js

```diff
@@ -127,6 +127,16 @@ export default {
       ...PASSWORD_FORM_FIELDS,
     ],
   },
+  password_forgot: {
+    fields: [
+      {
+        name: "email",
+        title: t`Email address`,
+        placeholder: "The email you use for your Metabase account",
```

Comment:
I believe we should translate the placeholder too

Reply:
Thanks, fixed!